app.py

```diff
@@ -13,10 +23,31 @@ def hello_world():
 def test():
     return 'Hello, test!'
 
-@app.route('/getData', methods = ['GET'])
-def getData():
-	return json.dumps({"success": True,
-  "events": [ {"chapter_number": 1, "section_number": 1, "user_id": 333, "date": datetime.datetime.now().isoformat()}, {"chapter_number": 1, "section_number": 1, "user_id": 444}]})
+def getEvents(zybook_code, user_id):
+	query = "SELECT event FROM user WHERE zybook_code=" + "'" + zybook_code + "'"
+	cur.execute(query)
+	rows = cur.fetchall()
+	user_rows = []
+	print(user_id)
+	for elem in rows:
+		if json.loads(elem[0])['event']['user_id'] == int(user_id):
+			user_rows.append(json.loads(elem[0])['event'])
+
+	return user_rows
+
+@app.route('/zybook/<zybook_code>/events/<user_id>', methods = ['GET'])
+def getData(zybook_code, user_id):
```

Comment:
Let's rename this method to something more descriptive like `get_user_zybook_events`